charts/portworx/values.yaml

```diff
@@ -26,9 +26,10 @@ customRegistryURL:
 registrySecret:
 
 lighthouse: true
-lighthouseVersion: 2.0.1
+lighthouseVersion: 2.0.2
 lighthouseSyncVersion: 0.3
 lighthouseStorkConnectorVersion: 0.1
+pxCentral: false
```

Comment:
let's use the same param names as install.portworx.com. So let's use `monitoring`.